api/checks/azure_test.go

```diff
@@ -0,0 +1,79 @@
+package checks
```

Comment:
You forgot the "small" tag. (And Travis should complain; if not, we have a problem.)

Reply:
Done.